gno.land/cmd/gnoland/start_test.go

```diff
@@ -116,7 +116,7 @@ func TestStart_Lazy(t *testing.T) {
 	io.SetErr(commands.WriteNopCloser(mockErr))
 
 	// Create and run the command
-	ctx, cancelFn := context.WithTimeout(context.Background(), 10*time.Second)
+	ctx, cancelFn := context.WithTimeout(context.Background(), 20*time.Second)
```

Comment:
I organically hate this test :)

Reply:
As long as it's organic 🥬 ;)

Reply:
I will try to restore the master value on this with opti